hw2/05-form.html

```diff
@@ -8,12 +8,58 @@
     <link
       rel="stylesheet"
       href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css" />
+      <link
+      rel="stylesheet"
+      href="styles.css"/>
+      <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/js/bootstrap.bundle.min.js"></script>
     <script defer src="05-form.js"></script>
   </head>
   <body>
     <main>
       <form>
```

Comment:
Adding a legend and a fielded is a recommended thing to do to organize the form.